src/web/web-service/ot_client.cpp

```diff
@@ -41,6 +41,7 @@
 #include <string.h>
 #include <sys/socket.h>
 #include <sys/un.h>
+#include <sys/select.h>
```

Comment:
nit: our coding style requires the include ordering to be alphabetical
```suggestion
#include <sys/select.h>
#include <sys/socket.h>
#include <sys/un.h>
```